README.md

```diff
@@ -85,10 +85,10 @@ Options:
      - Enumeration values that don’t match the value used by the service.
 
      Model validation _requires_ example payloads (request/response) of the service, so the data can be matched with the defined models. See [x-ms-examples extension](https://github.com/Azure/azure-rest-api-specs/issues/648) on how to specify the examples/payloads. Swagger “examples” is also supported and data included there is validated as well. To get the most benefit from this tool, make sure to have the simplest and most complex examples possible as part of x-ms-examples.
-     - Please take a look at the redis-cache swagger spec as an example for providing "x-ms-examples" over [here](https://github.com/Azure/azure-rest-api-specs/blob/master/arm-redis/2016-04-01/swagger/redis.json#L45).
-     - The examples need to be provided in a separate file in the examples directory under the api-version directory `azure-rest-api-specs/arm-<yourService>/<api-version>/examples/<exampleName>.json`. You can take a look over [here](https://github.com/Azure/azure-rest-api-specs/tree/master/arm-redis/2016-04-01/examples) for the structure of examples.
+     - Please take a look at the redis-cache swagger spec as an example for providing "x-ms-examples" over [here](https://github.com/Azure/azure-rest-api-specs/blob/389d3c2b3256fa48e3d78e07769907857b215536/specification/redis/resource-manager/Microsoft.Cache/stable/2016-04-01/redis.json#L68).
+     - The examples need to be provided in a separate file in the examples directory under the api-version directory `azure-rest-api-specs/arm-<yourService>/<api-version>/examples/<exampleName>.json`. You can take a look over [here](https://github.com/Azure/azure-rest-api-specs/tree/main/specification/redis/resource-manager/Microsoft.Cache/stable/2016-04-01/examples) for the structure of examples.

```

Comment:
```suggestion
     - The examples need to be provided in a separate file in the examples directory under the api-version directory `azure-rest-api-specs/arm-<yourService>/<api-version>/examples/<exampleName>.json`. You can take a look over [here](https://github.com/Azure/azure-rest-api-specs/tree/1c100daabed6a008c76f3e4e0cae4e7eac5a5bf4/specification/redis/resource-manager/Microsoft.Cache/stable/2016-04-01/examples) for the structure of examples.

```

You're absolutely right to call out the fact that `main` rots over time. I recently took over maintenance of this repo, and I agree we should be pinning these references by commit.